public_html/js/index.js

```diff
@@ -1,104 +1,113 @@
+				alert( 'There was an error in connecting to database.' );
+				$( buttonId ).addClass( 'btn-' + buttonClass ).removeClass( 'btn-' + buttonClass + '-clicked' ).blur();
+				$( unusedButtonId ).removeClass( 'btn-secondary' ).prop( 'disabled', false ).addClass( 'btn-' + unusedButtonClass );
 			}
-			$( compareDiv ).addClass( 'copyvios-fetched' );
 		} );
 	}
-}
-
 
-function loadMoreResults() {
-	$( '#btn-load-more' ).text( '' ).addClass( 'btn-loading' );
-	var lastId = $( '.ithenticate-id:last' ).text();
-	console.log( lastId );
-	$.ajax( {
-		url: 'loadmore',
-		data: {
-			lastId: lastId
+	/**
+	 * Open the compare pane and do an AJAX request to Copyvios to fetch comparison data
+	 * @param id Ithenticate ID of record
+	 * @param index Index of the link in the copyvios list for the record
+	 * @param copyvio Copyvio URL
+	 * @param diffId Oldid of diff
+	 */
+	function toggleComparePane( id, index, copyvio, diffId ) {
+		var compareDiv = '#comp' + id + '-' + index;
+		$( compareDiv ).slideToggle( 500 );
+		if ( !$( compareDiv ).hasClass( 'copyvios-fetched' ) ) {
+			$.ajax( {
+				type: 'GET',
+				url: 'https://tools.wmflabs.org/copyvios/api.json',
+				data: {
+					oldid: diffId,
+					url: copyvio,
+					action: 'compare',
+					project: 'wikipedia',
+					lang: 'en',
+					format: 'json',
+					detail: 'true'
+				},
+				dataType: 'json',
+				jsonpCallback: 'callback'
+			} ).done( function ( ret ) {
+				console.log( 'XHR Success' );
+				if ( ret.detail ) {
+					// Add a class to the compare panel once we fetch the details to avoid making repetitive API requests
+					$( compareDiv ).find( '.compare-pane-left' ).html( ret.detail.article );
+					$( compareDiv ).find( '.compare-pane-right' ).html( ret.detail.source );
+				} else {
+					$( compareDiv ).find( '.compare-pane-left' ).html( '<span class="text-danger">Error! API returned no data.</span>' );
+					$( compareDiv ).find( '.compare-pane-right' ).html( '<span class="text-danger">Error! API returned no data.</span>' );
+				}
+				$( compareDiv ).addClass( 'copyvios-fetched' );
+			} );
 		}
-	} ).done( function ( ret ) {
-		$( '#btn-load-more' ).text( 'Load More' ).removeClass( 'btn-loading' );
-		$newRecords = $( ret ).find( '.record-container' );
-		console.log( $newRecords );
-		$( '.record-container' ).append( $newRecords.html() );
-	} );
-}
+	}
+
+
+	function loadMoreResults() {
+		$( '#btn-load-more' ).text( '' ).addClass( 'btn-loading' );
+		var lastId = $( '.ithenticate-id:last' ).text();
+		console.log( lastId );
+		$.ajax( {
+			url: 'loadmore',
+			data: {
+				lastId: lastId
+			}
+		} ).done( function ( ret ) {
+			$( '#btn-load-more' ).text( 'Load More' ).removeClass( 'btn-loading' );
+			$newRecords = $( ret ).find( '.record-container' );
+			console.log( $newRecords );
+			$( '.record-container' ).append( $newRecords.html() );
+		} );
+	}
+} );
```

Comment:
This whole file is just wrapping the JS in `$(document).ready` and adding the necessary listeners at the top